modules/tensor_mechanics/src/materials/ComputeBeamEigenstrainFromVPP.C

```diff
@@ -0,0 +1,112 @@
+//* This file is part of the MOOSE framework
```

Comment:
Since it's not coming straight from a VPP, I think a name like "ComputeBeamEigenstrainFromCSVInterpolator" would be more clear (assuming you rename VectorPostprocessorToInterpolator to CSVInterpolator).